src/cfg/mod.rs

```diff
@@ -0,0 +1,98 @@
+use docopt::Docopt;
+use slog::Logger;
+use std::fs::File;
+use std::io::Read;
+use toml::from_str;
+
+pub mod errors {
+    error_chain!{}
+}
+
+use self::errors::*;
+
+const CONFIGFILE: &'static str = "blacklung.cfg";
```

Comment:
Yes, we should have an internal default value which can be overridden with a cfg file value which in turn can be overridden with a command line argument.